.appveyor.yml

```diff
@@ -1,6 +1,8 @@
 ﻿version: 2.3.2.{build}
 
-image: Visual Studio 2015
+image:
+  - Visual Studio 2015
+  - Visual Studio 2019
```

Comment:
I think `APPVEYOR_BUILD_WORKER_IMAGE` can be used to simplify config and get rid of `matrix exclude`.

https://www.appveyor.com/docs/build-environment/#using-multiple-images-for-the-same-build

Reply:
WOW I did not know that! Ok let me try, that would be a lot easier to read and manage than using `matrix exclude`